packages/grid/x-data-grid/src/components/virtualization/GridVirtualScroller.tsx

```diff
@@ -1,53 +1,105 @@
 import * as React from 'react';
```

Comment:
@romgrk Do you have an idea why the vertical scroll thumb size has changed in some cases?

https://app.argos-ci.com/mui/mui-x/builds/16813/70843502
https://app.argos-ci.com/mui/mui-x/builds/16813/70843559

Reply:
I think it's supposed to have changed in all cases, because the headers are now inside the virtual scroller's scroll-container, so they count towards the scroll content's height despite being fixed at the top.

Reply:
Yes, this makes sense to me.
Any idea why it didn't change in the second demo?

Reply:
Oh, I think it didn't change because there are 100 rows in the second demo, and the impact of one additional row (header) wasn't enough to increase the scroll thumb height even by 1px. Does this make sense?

Reply:
Yes 100 rows and more demos don't seem to change.